### ISSUE TITLE
add clipboard manager to default Cinnamon setup

### ISSUE THREAD

**Issue by alexanderdd** [linuxmint/cinnamon#9192]
```
 * Cinnamon version 4.4.8
 * Distribution Linux Mint 19.3 Cinnamon
more: https://termbin.com/hjxyz

**Issue**
clipboard manager not installed by default

**Steps to reproduce**
1. Install System with Cinnamon
1. to gain clipboard manager you have to know it exists, and install it.

**Expected behaviour**
Clipboard manager is installed by default, running, visible as indicator in system tray.

**Other info**
* having a clipboard manager installed by default will give cinnamon a distinguishing advantage over windows and other linux DEs. 
* Cinnamon strives to be the most user friendly DE? Accidentally deleting something from your clipboard by copying something else is very common and really annoying. But most people dont know clipboard managers exist.
* I dont care which one you choose. I like clipit, but diodon works fine too.
* this program is tiny so it is not wasting resources at all

**Comment by joaoherberto** [comment]
+1 for this 

BUT WITH ONE ADDITION! Add a Contextual Menu item displaying the last 5 to 10 items in the clipboard to quickly paste any of those items.

As an example, say you are working on a document, a user would "Right Click" on document area, select "Paste from clipboard" and it would show all the items that can be pasted, the user selects the item and it is pasted.

I currently use Parcellite and it is fine, but having to constantly going to the top menu to switch which item I want to re-paste seems repetitive, a simple right click and paste seems logical.

**Comment by leigh123linux** [comment]
Maybe something like this could be ported.

https://extensions.gnome.org/extension/779/clipboard-indicator/

**Comment by jvschiavo** [comment]
I was just searching for one now. Btw, does anyone have a good suggestion for a clipboard manager for Mint Cinnamon?

**Comment by joaoherberto** [comment]
I use clipit

**Comment by rastla** [comment]
I agree that it should be included by default, but:

> having a clipboard manager installed by default will give cinnamon a distinguishing advantage over windows and other linux DEs.

Windows 10 even has a clipboard manager by default (Win+V) since Update 1809 I believe

**Comment by claudiux** [comment]
Why not `diodon`?

**Comment by SemenMartynov** [comment]
+1
It would be great to have this feature by default.

**Comment by tdemin** [comment]
@joaoherberto ClipIt doesn't handle images.

**Comment by alexanderdd** [comment]
should we make a vote? Here or in the linuxmint forums? I don't know how decisions are made around here...

**Comment by bernd-wechner** [comment]
> Why not `diodon`?

To be honest because I just tried it and it doesn't work. Nor does CopyQ haha. I guess Id up this feature request to ad one that works properly. I have used CopyQ for years and love it, but the problem is and why I'm looking now is that it doesn't catch WebAPI copy commands properly. Sometimes it does, sometimes not sometimes half (the text/plain part yes the text/html part no).

diodon I just tried and noticed nothing in that space. So CopyQ is one up on it.

But that aside, given Mint's standing as the install and go distro, to be honest I think a default clipboard manager installed from the word go would rock.

**Comment by kendal-pioneer** [comment]
As a KDE user too. I have to agree. It is really needed

**Comment by jwmh** [comment]
+1

**Comment by OseiasRocha** [comment]
+1

**Comment by abhitrueprogrammer** [comment]
+1

**Comment by abhitrueprogrammer** [comment]
> > Why not `diodon`?
> 
> To be honest because I just tried it, and it doesn't work. Nor does CopyQ haha. I guess I'd up this feature request to add one that works properly. I have used CopyQ for years and love it, but the problem is and why I'm looking now is that it doesn't catch WebAPI copy commands properly. Sometimes it does, sometimes not, sometimes half (the text/plain part yes the text/html part no).
> 
> diodon, I just tried and noticed nothing in that space. So CopyQ is one up on it.
> 
> On top of that, Gpaste, Parcellite, ClipIt and Keepboard only manage the text/plain parts and can't see the text/html part or images. Pffft. CopyQ still reigns (even though it's a bit flaky with the WebAPI copy command).
> 
> But that aside, given Mint's standing as the install and go distro, to be honest, I think a default clipboard manager installed from the word go would rock.

I tried copyQ just now and the app seems to work well? Either way I think it's a absolute must  for mint to have a pre-installed app/applet for a feature like this. KDE has it and Gnome supports it through Pano. Even Windows and Android have their respective clipboard managers, while we don't really have anything close for mint. Would love to have something pre-installed.

**Comment by KOT040188** [comment]
I recently switched to cinnamon and was very surprised that it does not have a clipboard manager.  I've been searching for a week now, but haven't found anything good.

**Comment by delinuxco** [comment]
Install Gpaste and  Gpaste applet

**Comment by KOT040188** [comment]
Yes, I tried that.  But it has shortcomings.  I could not configure the mouse buffer (primary) in it so that it would not be cleared when the application was closed.  Also I would like to pin some common text fragments for pasting (there are many extensions for gnome with this feature}.

**Comment by bernd-wechner** [comment]
I'd see it as a + if Mint took on CopyQ as one of its Xapps. I see solid clipboard management as a huge feather in the cap of any wanna be top slick DE,

**Comment by hthienloc** [comment]
+1

**Comment by gVesco** [comment]
+1

**Comment by JosephMcc** [comment]
After 6 years, this is unlikely to happen any time soon. In the future all feature requests go here: https://github.com/orgs/linuxmint/discussions
